src/taco-dependency-installer/dependencyInstaller.ts

```diff
@@ -355,14 +372,14 @@ module TacoDependencyInstaller {
 
             return installerUtils.promptUser(resources.getString("YesExampleString"))
                 .then(function (answer: string): Q.Promise<any> {
-                    if (answer === resources.getString("YesString")) {
+                    if (answer.toLocaleLowerCase() === resources.getString("YesString")) {
```

Comment:
That makes sense! What do you think about creating our own "utlity" method to ask for [y/n] so all the modules can use that same function?